### ISSUE TITLE
Unified rendering of healthcare objects

### ISSUE THREAD

**Issue by kvlahromei** [gravitystorm/openstreetmap-carto#2938]
Hi,

currently amenity=doctors, amenity=hospital, shop=optican, ... are covered in the current style.  
I recommend to add further [healthcare=* objects](https://wiki.openstreetmap.org/wiki/Key:healthcare) as there are esp. in big cities more complex clinic / campus areas that would benefit from a more detailed (but also balanced) rendering of that institutions.

For example [here in Rostock](http://www.openstreetmap.org/#map=17/54.08443/12.10241) we are currently hiding this details
![osm rostock schillingallee](https://user-images.githubusercontent.com/8177370/32557057-5efb4318-c4a1-11e7-9dfc-c8a8c3488a16.png).

It would be great, if we can present this POIs in the same pink tone (but without icons) as the doctors, veterinarian, ...

(This was already suggested in #1981, #763 some years ago. But I see a growing popularity / more precise tagging since that time)

**Comment by HolgerJeromin** [comment]
Looking at the list i could imaging some (birthing_center, midwife, blood_donation, physiotherapist, psychotherapist, rehabilitation) can have a healthcareColor dot. But not everything (for example blood_bank is no place to go).

This issue could get an meta issue like #108

**Comment by kocio-pl** [comment]
This one could be meta, you just need to make a list. :smile:

**Comment by Matthias84** [comment]
Ok, so listing what needs to be done:

**new**, with pink cross/dot and maybe with name=* or operator=* label for healthcare=...
- [ ] yes
- [ ] birthing_center
- [ ] blood_bank
- [ ] blood_donation
- [ ] centre
- [ ] hospice
- [ ] rehabilitation

**conflicting ones**, they are covered by some other tags
- [ ] clinic  - amenity=clinic
- [ ] hospital - amenity=hospital
- [ ] dentist - amenity=dentist
(following ones by a more general amenity=doctors)
- [ ] audiologist
- [ ] alternative
- [ ] midwife
- [ ] occupational_therapist
- [ ] optometrist
- [ ] physiotherapist
- [ ] podiatrist
- [ ] psychotherapist
- [ ] speech_therapist

(just give me a ping, if you prefer an sep. issue ticket instead or if I embed it to the first post ...)

IMHO it's worth to tune the styling for zoomlevels. For example hospitals are more important than centres or doctors. 

_BTW: I also started an [JOSM preset](https://josm.openstreetmap.de/wiki/Presets/Healthcare)_ ...

**Comment by kocio-pl** [comment]
Separate tickets would be needed later, I think first we need to know the numbers of uses for each case.

**Comment by Matthias84** [comment]
@kocio-pl  Of course, I updated the post above and included the usage provided by taginfo.

Just that you don't get me wrong: I don't want to oppress the healthcare schema in favour of the traditional tags. Instead I like to enable the support, so mappers can also use the healthcare tagging to support new features as midwifes, centres, ... which IMHO would allow an better styling, even if the community starts micromapping every single practitioner and therapist within a city 😉

P.S. Be aware, there are some other "lost ones" 10k objects tagged for example with [health_facility=*](https://taginfo.openstreetmap.org/keys/health_facility%3Atype).

**Comment by Matthias84** [comment]
Is there anything left I can do to speedup the development of this feature? :smiley:

**Comment by kocio-pl** [comment]
Making a PR and rendering examples would certainly be a progress.

**Comment by Slawek234** [comment]
I prepared a sample icon for the laboratory. https://commons.wikimedia.org/wiki/File:Mikroskop9.png

**Comment by kocio-pl** [comment]
How would it look like in amenity brown or healthcare red in 14 px matrix? I believe that shape is too complex for this size.

**Comment by Slawek234** [comment]
pink https://s9.postimg.org/3od4ph8bz/pink_14.png
red https://s9.postimg.org/qd2bp42lb/red_14.png
brown https://s9.postimg.org/z835zktxr/brown_14.png
black https://s9.postimg.org/y5szh2lf3/black_14.png

**Comment by kocio-pl** [comment]
It's still 64 px and the last one is too simple, indeed.

**Comment by Slawek234** [comment]
now they have a max. 14pix
pink https://s9.postimg.org/c7winh91b/pink_14.png
red https://s9.postimg.org/l2xcy05jj/14_red.png
brown https://s9.postimg.org/y8cv3q9lb/14_brown.png
black https://s9.postimg.org/dbgmz5r0v/14_black.png

**Comment by polarbearing** [comment]
Not sure if wikimedia commons wants to get cluttered with lots of design drafts. 
You could use the _gist_ function her in github.

**Comment by Tomasz-W** [comment]
What do you think about change health objects colour to red? For me, it would fit better (e.g. red crosses on ambulances   https://www.google.pl/search?client=opera&hs=6a0&biw=1920&bih=981&tbm=isch&sa=1&ei=w3qFWqL4GZLfwQLcyJeQAg&q=ambulance&oq=ambulance&gs_l=psy-ab.12...0.0.0.1115112.0.0.0.0.0.0.0.0..0.0....0...1c..64.psy-ab..0.0.0....0.cbuGlLSwW1w )
If we are considering dot-rendering for some of these objects, it would be good to distinguish them more from violet shop=* dot.

**Comment by kocio-pl** [comment]
I would have to see the test rendering to have any idea about it.

**Comment by Tomasz-W** [comment]
I've made few simple mockups in Photoshop. At the bottom center I've added triangle with 3 dots for comparsion: current health-pink on the left, red on the right and shop=* violet at the bottom.
(click to view full size!)

Current pink, zoom 18
![1 1](https://user-images.githubusercontent.com/25656654/36267924-7981931e-1275-11e8-8f76-846d7ebec3a4.jpg)
Red, zoom 18
![1 2](https://user-images.githubusercontent.com/25656654/36267925-799db4ea-1275-11e8-87a3-cbbbde212655.jpg)
Current pink, zoom 19
![2 2](https://user-images.githubusercontent.com/25656654/36267926-79b8f98a-1275-11e8-8867-dc71766b182a.jpg)
Red, zoom 19
![2 1](https://user-images.githubusercontent.com/25656654/36267927-79d3b630-1275-11e8-82f7-a86294b04eb2.jpg)

**Comment by kocio-pl** [comment]
Thanks! I don't feel the urge to change it, so that's up to the rest of community, including preparing PR, but it seems OK and it may make it easier to distinguish.

**Comment by Matthias84** [comment]
I recommend to start an separated issue, as this is related to healthcare, but isn't tied to this extension of further healthcare objects itself.

BTW: The colors are tied to the culture. While people from Europe / East know red for emergencies, users from America search for pink/green icons https://wiki.openstreetmap.org/w/index.php?title=Proposed_features/Healthcare&oldid=1318635#Rendering

**Comment by Slawek234** [comment]
> The colors are tied to the culture. 
In the case of a hospital (in my opinion the most-important (but not the most popular) health care element) I can not agree in the Christian & Muslim civilization health care have red colour (Maybe in other civilizations it is different).
I can give the info that the liquid tissue of warm-blooded vertebrates is red.

However, there is a lot of diversity regarding pharmacies! 
In Poland, the symbols of pharmacies are usually red or green, although they also happen blue.

Please look https://en.wikipedia.org/wiki/Comparison_of_European_road_signs
Hospital/First aid and Pharmacy.

**Comment by Tomasz-W** [comment]
@Matthias84 
Refering to https://github.com/gravitystorm/openstreetmap-carto/issues/2938#issuecomment-343666041 , I think that only the tags which are 1:1 to certain amenity=* tag (e.g. amenity=dentist and healthcare=dentist) should had the same icons, and all the rest should be rendered as red dot with a label.

**Comment by Slawek234** [comment]
Is there any chance that healthcare could be in different colors, for example:
- Hospital icon red
- doctors, clinics, something similar to what is now pink
- pharmacy icon green color

Unless it would not be consistent?

**Comment by Tomasz-W** [comment]
I think these objects should be unified by one color.

**Comment by kocio-pl** [comment]
I also think this should be one color. Green is used for leisure/sport, so don't belong here, but red or current dark pink would be OK for me.

**Comment by Tomasz-W** [comment]
Sorted list:

_Separate icons already:_
* healthcare=clinic   ->   amenity=clinic
* healthcare=dentist   ->   amenity=dentist
* healthcare=doctor   ->   amenity=doctors
* healthcare=hospital   ->   amenity=hospital
* healthcare=pharmacy   ->   amenity=pharmacy

_Dot-rendering candidates list:_
* healthcare=alternative
* healthcare=audiologist 
* healthcare=birthing_center
* healthcare=blood_bank
* healthcare=blood_donation
* healthcare=centre
* healthcare=laboratory 
* healthcare=midwife 
* healthcare=occupational_therapist
* healthcare=optometrist
* healthcare=physiotherapist
* healthcare=podiatrist
* healthcare=psychotherapist
* healthcare=rehabilitation
* healthcare=speech_therapist
* healthcare=yes

**Comment by Slawek234** [comment]
I looked at it quickly https://wiki.openstreetmap.org/wiki/Key:healthcare

"Someone who practices podiatry but is not a medical practitioner." Is it possible, is it legal? A few years ago, I heard something like parents believed a man who, not being a doctor, claimed that by starving a child he would cure them ... The child died of hunger, parents were locked in jail, the other children were taken away, the doctor searched for the police, I do not know if they found.
I think there are mixed-up paramedical & medical services together I can not distinguish them easily.

Regarding healthcare=optometrist "Optometrist; If linked with a shop please use shop=optician instead"
It looks like it is healthcare=doctor -> amenity=doctors or shop=optican

Regarding healthcare=midwife | Midwife.
It looks like it is healthcare=doctor -> amenity=doctors

The amenites are combined with the medicine that I know we have 21st century:
    healthcare=clinic -> amenity=clinic
    healthcare=dentist -> amenity=dentist
    healthcare=doctor -> amenity=doctors
    healthcare=hospital -> amenity=hospital
    healthcare=pharmacy -> amenity=pharmacy

This list below is to improve the definition on the wiki because they are partly duble with amenity = doctors partly with alternative?
    healthcare=alternative
    healthcare=audiologist
    healthcare=birthing_center
    healthcare=blood_bank
    healthcare=blood_donation
    healthcare=centre
    healthcare=chiropractor
    healthcare=laboratory
    healthcare=midwife
    healthcare=occupational_therapist
    healthcare=optometrist
    healthcare=physiotherapist
    healthcare=podiatrist
    healthcare=psychotherapist
    healthcare=rehabilitation
    healthcare=speech_therapist
    healthcare=yes

 Why should it be omitted healthcare=laboratory? 
Here is a list of these entities for Poland http://kidl.org.pl/index.php?page=lista-laboratoriow
Each developed country has its own list of certified laboratories. Great Britain, Germany, France, USA, Spain can be exchanged for a long time, etc.
It is subject to similar restrictions as the medical practice

**Comment by HolgerJeromin** [comment]
> Regarding healthcare=midwife | Midwife.
It looks like it is healthcare=doctor -> amenity=doctors medical specialization Midwife

Midwifes are no doctors. At least not in germany

**Comment by Slawek234** [comment]
@HolgerJeromin You're right, I've translated it wrong.

**Comment by dieterdreist** [comment]
2018-02-20 23:23 GMT+01:00 Slawek234 <notifications@github.com>:

> "Someone who practices podiatry but is not a medical practitioner." Is it
> possible, is it legal?
>


it doesn't matter for documenting tags and mapping if it is legal or not
and in which jurisdiction. If you don't know about such places, simply
don't use the tag. If you know about such places and know they are illegal
in your context, decide yourself what to do.

**Comment by Tomasz-W** [comment]
@andrzej-r 
As #3163 is ready to merge, are you interested in write code for this issue? It's very simillar - it includes adding dot rendering for tags from https://github.com/gravitystorm/openstreetmap-carto/issues/2938#issuecomment-366990786 and (optionally) change healthcare colour to red.

**Comment by andrzej-r** [comment]
I will look into it after #3163 is done.

**Comment by kocio-pl** [comment]
Are you still planning to prepare a code for this issue?

**Comment by andrzej-r** [comment]
I haven't looked into it yet (bug fixes). How much time do we have to the release?

**Comment by kocio-pl** [comment]
I plan to make it on 11.05 (see #3207), but I don't think it's important here. Such a big change as adding the whole key may take some time to discuss and test, as you've seen witf `office=*`. Just do it whenever you feel like.

**Comment by Tomasz-W** [comment]
@andrzej-r As #3163 is merged and working well, are you still interested in write a PR for this issue?

**Comment by kocio-pl** [comment]
It might be better to NOT use red for healthcare to avoid problems with Red Cross:

https://lists.openstreetmap.org/pipermail/talk/2018-July/081050.html

**Comment by Tomasz-W** [comment]
Uh, I hate patents etc. so much...

**Comment by Tomasz-W** [comment]
I'm curious would it be still a problem if we would chande hospital icon to this shape?
![cross-hospital-first-aid-doctor-medical-health-3980fbb6a739ff5d-512x512](https://user-images.githubusercontent.com/25656654/43649841-8837585a-973e-11e8-8cf4-1a3f84ddfb19.png)

There is no "red cross" then.

**Comment by kocio-pl** [comment]
The idea is nice (clever hack!), but one has to test if it wouldn't be too strong and that would break similarity to other health objects.

**Comment by Tomasz-W** [comment]
Current icon:
![pobrany plik](https://user-images.githubusercontent.com/25656654/43654819-d637dbfc-974c-11e8-85a8-56d1c31696ef.png)

Icon proposal:
![amenity hospital](https://user-images.githubusercontent.com/25656654/43655063-d7439094-974d-11e8-882c-2927235f54ee.png) ![amenity hospital](https://user-images.githubusercontent.com/25656654/43655075-df4fe4ea-974d-11e8-875f-4e08607aee38.png)

Gist link: https://gist.github.com/Tomasz-W/d9e415d98d201270efd45ab59156b2a5

**Comment by meased** [comment]
I like this icon better than the current one, though icons with too much "empty space" in them may not look good on complicated backgrounds.

Maybe off topic, but I think icons need a halo like text has. Or maybe all icons should be shielded (in a circle/rectangle with a foreground and background color so they're readable regardless of the background).

**Comment by Tomasz-W** [comment]
Just for the note: if someone is going to make a test renderings for healthcare-red, please try with #DA0000

**Comment by polarbearing** [comment]
The white cross on red is nice, but it's the flag of Switzerland:
<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/08/Flag_of_Switzerland_%28Pantone%29.svg/160px-Flag_of_Switzerland_%28Pantone%29.svg.png">

**Comment by kocio-pl** [comment]
The flag has a very different shape - it's square, not a circle, so I have no problem with that.

**Comment by dieterdreist** [comment]
a cross is generally refering to the christian culture, and to the red
cross, obviously. Using the swiss flag is misleading as well.
a circle and a square are not "very different", they are essentially the
same.

**Comment by jragusa** [comment]
Following the suggestions of @Tomasz-W 
https://www.openstreetmap.org/#map=19/46.18687/6.22480

before
![healthcare_color_before](https://user-images.githubusercontent.com/9897203/45647786-0af6ba00-bac7-11e8-8b2e-89c8c4535a08.png)

new colour DA0000
![healthcare_color_after](https://user-images.githubusercontent.com/9897203/45647819-21047a80-bac7-11e8-90d7-de9f13f05255.png)

new colour + icon
![healthcare_color icon_after](https://user-images.githubusercontent.com/9897203/45647830-2a8de280-bac7-11e8-865f-d3007d9249e7.png)

**Comment by meased** [comment]
Looks good. I think the new icon is a little too light and would look better with the border thickened up a bit like this:

![health](https://user-images.githubusercontent.com/14190496/45650031-dad8f800-ba81-11e8-9dc0-414439ca550f.png)

https://gist.github.com/meased/71eee9c8bec20b07d44d73539fe33943

**Comment by jragusa** [comment]
new colour + icon from @meased 

![healthcare_color icon2_after](https://user-images.githubusercontent.com/9897203/45674623-cbaf8400-bb2d-11e8-8d6e-4df3649417cf.png)

**Comment by Matthias84** [comment]
Beside the design aspect, is anybody already working on a style patch that introduces the mendtioned tags at my first post?

**Comment by Tomasz-W** [comment]
@jragusa Are you willing to write a PR with generic dots for healthcare=* objects also?

**Comment by kocio-pl** [comment]
@Matthias84 From my experience any systematic tags rendering, no matter how obvious it looks, needs someone who will take care of it and make an overall algorithm before coding. Would you like to make more detailed rendering rules proposition so someone could code it?

I mean something like "we should render this and that the same as something, but this one is not clear so we should not render it now".

**Comment by jragusa** [comment]
@Tomasz-W I can take a look, but I will not be available next week.

**Comment by jragusa** [comment]
Ok, I would like to know what will be included into the PR ?

I suppose : 
* Replace colour (label and icon) to #DA0000
* Add new icon to `amenity=hospital`
* Render as dot the healthcare scheme (see [list](https://github.com/gravitystorm/openstreetmap-carto/issues/2938#issuecomment-366990786) of @Tomasz-W)

I would like to know what will be the strategy with duplicate healthcare tags of clinic, dentist, doctors, hospital and pharmacy ? Do we render as amenity equivalents ?

@Tomasz-W Do you want to add some new icons to the PR ?

**Comment by kvlahromei** [comment]
I supose only Render as dot the healthcare scheme, to get focused changerequest. To alter the healthcare design, is an seperated issue.

**Comment by kocio-pl** [comment]
Please make the changes more atomic, even if the code is quite simple, since it makes it easier to test. For example I still did not have the time to test #3318 fully.

I guess z18+ is safer, since we don't know what they really are, because this scheme is very wide.

**Comment by kocio-pl** [comment]
From a quick reading it makes sense for me. Red/pink dot will be recognizable enough.

**Comment by jragusa** [comment]
Is there a specific way to consider new column added in projet.mml ?

When I'm trying rendering, I get the following error:

`Postgis Plugin: ERROR:  column "healthcare" does not exist`

Yes, I damned with PostGIS...

**Comment by kocio-pl** [comment]
We have hstore for such features, see for example how `seasonal` key is used.

**Comment by Adamant36** [comment]
@kocio-pl, how do you know when its better to create a new column for something and when to use hstore versus not?

**Comment by kocio-pl** [comment]
This has been already defined in database pre-processing step, look here:

https://github.com/gravitystorm/openstreetmap-carto/blob/master/openstreetmap-carto.style

**Comment by meased** [comment]
I don't know which is more efficient, but without hstore I believe you can do:

``` sql
    tags->'healthcare' as healthcare,
```

**Comment by kocio-pl** [comment]
It's not about efficiency. Changes in `openstreetmap-carto.style` are possible, but require database reload, so we avoid it, because it creates problems in deployment. There might be a moment where we convert some hstore values into the proper database tables, but this should not be too frequent.

**Comment by jragusa** [comment]
Ok, I resolved the issue

https://www.openstreetmap.org/way/217598504 (polygon)
before
![healthcare_polygon_before](https://user-images.githubusercontent.com/9897203/47220530-5942f580-d3b2-11e8-8566-9fe063f50dae.png)

after
![healthcare_polygon](https://user-images.githubusercontent.com/9897203/47220398-f8b3b880-d3b1-11e8-8844-2e9b2bc756a1.png)

https://www.openstreetmap.org/node/5799816033 (node)
before
![healthcare_node_before](https://user-images.githubusercontent.com/9897203/47220539-5e07a980-d3b2-11e8-898f-c3af1dccde8e.png)

after
![healthcare_node](https://user-images.githubusercontent.com/9897203/47220418-0701d480-d3b2-11e8-85c8-fd8db8e49a10.png)

**Comment by Tomasz-W** [comment]
@jragusa Thanks! Anyway, I would like to see some "wider" examples (maybe on lower zooms?) to rate if this colour isn't too prominent comparing to the rest of map.

**Comment by jragusa** [comment]
z19
![healthcare_z19](https://user-images.githubusercontent.com/9897203/47226551-22281080-d3c1-11e8-87c7-6eb693da1dd4.png)

z18
![healthcare_z18](https://user-images.githubusercontent.com/9897203/47226416-c2316a00-d3c0-11e8-922e-167a85b38191.png)

z17
![healthcare_z17](https://user-images.githubusercontent.com/9897203/47226433-ccebff00-d3c0-11e8-86e8-0ed6aecb457b.png)

z16
![healthcare_z16](https://user-images.githubusercontent.com/9897203/47226451-d83f2a80-d3c0-11e8-99bf-10acd82fe088.png)

**Comment by Tomasz-W** [comment]
My remarks:
* I think it's a little bit too bright and we can make it minimally darker -> `#CC0000`
* push `amenity=dentist` icon to z18 (with earlier dot rendering)
* consider pushing initial zoom level for `amenity=clinic/ doctors/ veterinary` icons
* consider pushing `amenity=hospital` initial zoom level to z16 (the only one icons on z15 which I can see are castles and hospitals, but hospitals can be quite dense: https://www.openstreetmap.org/#map=15/48.8792/2.3806)

**Comment by jragusa** [comment]
Following the 2 first remarks
z17
![healthcare_z17_2](https://user-images.githubusercontent.com/9897203/47241923-a1cbd480-d3ed-11e8-99e0-ce0bcc01fa7c.png)

3/ Which zoom level do you propose for `amenity=clinic/doctors/veterinary` ?
4/ Based on the name tag, several of the hospitals displayed in the link above are incorrectly tagged and should use healthcare tag instead. Moreover it's a specific case and if you look at city of intermediate size, hospital is (probably?) the only one displayed at z15. See for example below the same location than previous pictures (the right one is also incorrectly tagged, it's a blood donation)
z15
![healthcare_z15](https://user-images.githubusercontent.com/9897203/47242317-a5ac2680-d3ee-11e8-9ef4-85992b930ffc.png)

**Comment by Tomasz-W** [comment]
@kocio-pl @meased @Adamant36 What do you think about current initial zoom levels of `amenity=hospital` and `amenity=clinic/doctors/veterinary`?

**Comment by polarbearing** [comment]
> hospitals can be quite dense

We might need to check with the French community about this, but there might be some facilities that are a clinic (day treatment) and not a hospital (overnight).

**Comment by Tomasz-W** [comment]
I made some more map checking and in the topic of zoom levels I propose:
* push `amenity=dentist` icon to z18 (with dot rendering at z17)
* left `amenity=hospital` as it is
* push `amenity=clinic/doctors/veterinary` icons to z17

**Comment by kocio-pl** [comment]
The levels were ready for doctors and hospital, we have added clinic between them, because we were not sure what is it really, but probably smaller than hospital and bigger than typical doctors. I think it was good decision.

> clinic (day treatment) and not a hospital (overnight)

Wow, sounds like a sane definition at last! Could you discuss it on Tagging and add on the wiki?

**Comment by polarbearing** [comment]
That was discussed on tagging a while ago and is reflected in the wiki:
- A hospital providing in-patient medical treatment ... outpatient care only should be tagged amenity=clinic
- Clinic: A medium-sized medical centre, typically without admission of inpatients.

**Comment by jragusa** [comment]
So what do you suggest for zoom level of `amenity=clinic/dentist/doctors/hospital/veterinary` ?

@kvlahromei, do you talk about `healthcare=clinic/dentist/doctors/hospital` ? 

They are currently not rendered in my preview. Do I include them too next to `amenity` equivalent ?

**Comment by kvlahromei** [comment]
Well, the intention of this ticket is to add support for healthcare=*
objects.

As later posts say, there are objects that are already covered by
amenity=* tags, but some new ones need to be added. Idealy they have the
same styling+zoom as existing health object classes.

(Personally I prefer not to mix altering the health styling and
introducing new hashtags. But of course, this is up to the PR author)


>>> Jérémy Ragusa <notifications@github.com> 22.10.2018 16:56 >>>
So what do you suggest for zoom level of
`amenity=clinic/dentist/doctors/hospital/veterinary` ?

@kvlahromei, do you talk about
`healthcare=clinic/dentist/doctors/hospital` ? 

They are currently not rendered in my preview. Do I include them too
next to `amenity` equivalent ?

**Comment by Tomasz-W** [comment]
@jragusa To be precisious: `amenity=doctors` and `veterinary `are currently rendered from z17 so there is no change needed anyway, but as `amenity=clinic` can be bigger or smaller, I think z17 would be more proper for it than a current one z16.

**Comment by meased** [comment]
> What do you think about current initial zoom levels of amenity=hospital and amenity=clinic/doctors/veterinary?

I don't have a problem with the current situation, but in the areas I'm familiar with, they are rather spread out.

**Comment by jragusa** [comment]
@Tomasz-W ok, I moved `amenity=clinic` so now clinic, doctors, pharmacy and veterinary are all displayed from z17

Anything else or can I create the PR ?

**Comment by Tomasz-W** [comment]
@jragusa That's all :)

**Comment by kvlahromei** [comment]
@jragusa Could you plz include the healthcare=* tag support for this amenities and the new introduced, too? (sorry if this is just an misunderstanding)

**Comment by jragusa** [comment]
@kvlahromei I missed your previous comment. I added them to the list. 

I take this opportunity to recommend a discussion in the tagging list about duplicated tag `healthcare` - `amenity`. I notice that several features with `healthcare=clinic/dentist/doctors/hospital` contain also their equivalent `amenity` tag.

**Comment by kvlahromei** [comment]
Thank you!

I pointed out this parallelism at 4th comment as the situation as it is currently in the wild. The healthcare proposal already [focused this migration](https://wiki.openstreetmap.org/w/index.php?title=Proposed_features/Healthcare&oldid=1318635#healthcare_versus_amenity) and suggests just to render both tagging schemas. After a while, the legacy tags might be cleaned up and forgotten.
